src/main/kotlin/com/terraformation/backend/i18n/Messages.kt

```diff
@@ -91,6 +91,16 @@ class Messages {
       NotificationMessage(
           title = "An accession has dried", body = "$accessionNumber has finished drying.")
 
+  fun nurseryBatchReadyNotification(
```

Comment:
Minor naming nit: this should be `seedlingBatchReadyNotification` or `nurserySeedlingBatchReadyNotification` since the full term is "seedling batch" rather than "nursery batch."

Reply:
Oh thank you for catching that. I was trying to be consistent with Nursery-Seedling-Batch everywhere but I missed this one.